jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskprocesscontext/TaskProcessContextPresenter.java

```diff
@@ -68,18 +77,23 @@ public TaskProcessContextPresenter(
             PlaceManager placeManager,
             Caller<TaskQueryService> taskQueryService,
             Caller<DataServiceEntryPoint> dataServices,
-            Event<ProcessInstancesWithDetailsRequestEvent> processInstanceSelected
+            Event<ProcessInstancesWithDetailsRequestEvent> processInstanceSelected,
+            ActivityManager activityManager
     ) {
         this.view = view;
         this.taskQueryService = taskQueryService;
         this.dataServices = dataServices;
         this.placeManager = placeManager;
         this.processInstanceSelected = processInstanceSelected;
+        this.activityManager = activityManager;
     }
 
     @PostConstruct
     public void init() {
         view.init(this);
+        final Set<Activity> activity = activityManager.getActivities(new DefaultPlaceRequest(PROCESS_INSTANCE_DETAILS));
+        enableProcessInstanceDetails = activity.isEmpty() == false;
```

Comment:
@cristianonicolai I'm sure you have actually tested this in ```kie-drools-wb``` but won't the ```ActivityBeansCache``` *always* contain "PROCESS_INSTANCE_DETAILS" if the Activity was on the classpath at GWT compile-time? I see your tests setup the ```ActivityManager``` to mock both scenarios but I fear it may not be the case at runtime. Can you share some thoughts to ease my paranoia? ;)